requirements/requirements.server.in

```
@@ -35,5 +35,6 @@ PyYAML==6.0.1                                               # globus_extras and
 globus-sdk==3.41.0                                          # globus_extras
 python3-saml==1.16.0                                        # saml_extras
 pymongo==4.8.0                                              # pymongo (metadata plugin)
+elasticsearch>=8.0.0                                        # elasticsearch ( metadata plugin)
```

```suggestion
elasticsearch==8.0.0                                        # elasticsearch (metadata plugin)
```

Server dependencies should always be pinned to a specific version, rather than providing a range; see https://rucio.github.io/documentation/developer/dependency_management/#in-and-txt-files

When pinning this dependency, you might want to change it to a later version instead (e.g. 8.15.1, like the one compiled in the `.txt` file)

---

pinned the dependency to 8.15.1 .